.github/workflows/release.yml

```diff
@@ -0,0 +1,39 @@
+name: Bump 'n' release
+
+on:  
+  release:
+    types: [published]
+
+jobs:
+  bump:
+    runs-on: ubuntu-latest
+    steps:
+      -
+        name: Checkout code
+        uses: actions/checkout@v2
+      - 
+        name: Set the new version inside the script
+        run: |
+          GIT_TAG=${{ github.event.release.tag_name }}
+          sed -i "s/^padd_version=.*/padd_version=\"$GIT_TAG\"/" padd.sh
+      - 
+        name: Create Pull Request to update script on master branch
+        uses: peter-evans/create-pull-request@v3
+        with:
+          base: master
+          commit-message: Update report
+          committer: Pralor <pralor-bot@users.noreply.github.com>
+          author: Pralor <pralor-bot@users.noreply.github.com>
+          branch: example-patches
+          delete-branch: true
+          title: 'Bump Script Version After Release'
+          body: |
+            Bumps the `padd_version=` variable in `padd.sh` to match the most recently released version
+          labels: |
+            internal          
+      -
+        name: Attach script to release
+        uses: softprops/action-gh-release@v1
```

Comment:
Doesn't this action create a new release?
The workflow is: publish a new release with tag. This triggers this action. It will open a PR with the changed version string  but also uploads the changed `padd.sh` as an asset to a (new?) release? How does it know to append the changed file to the just released release?

Also, as long as the PR is not merged, the asset `padd.sh` and the `padd.sh` on master will be out of sync I guess.

Reply:
This action only triggers on `release`, and as such it attaches to the calling release. It's the same action used in FTL here:

https://github.com/pi-hole/FTL/blob/c0cf8de918108bb3cda41d705a337665f614d588/.github/workflows/build.yml#L122-L127

The onus is on the releaser to merge the PR into `master` after it is created. But yeah, they are out of sync until the PR is merged. 

But we can also change all the install instructions to point at `https://github.com/pi-hole/padd/releases/latest/download/padd.sh` (And/or create a `padd.pi-hole.net` for that purpose)

Reply:
> Doesn't this action create a new release?

And no, it doesn't The release is created manually in the normal fashion (allowing us to manually decide a version number/add release notes etc) - and then the workflow is triggered